backend/tracim_backend/lib/utils/utils.py

```diff
@@ -218,7 +218,8 @@ def lighten(self):
 def string_to_list(
         base_string: str,
         separator: str,
-        cast_func: typing.Callable
+        cast_func: typing.Callable,
+        stripped: bool=False,
```

Comment:
stripped means "les espaces ont déjà été enlevés". what you want is `do_strip` : if true then string items will be stripped, else they won't). So rename it to `do_strip`

Reply:
done.